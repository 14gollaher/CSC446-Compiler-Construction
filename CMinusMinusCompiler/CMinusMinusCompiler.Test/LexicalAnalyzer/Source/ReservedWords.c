if else
while float
 int char
break	
	continue	void
if0else
0else
ifelse
if2