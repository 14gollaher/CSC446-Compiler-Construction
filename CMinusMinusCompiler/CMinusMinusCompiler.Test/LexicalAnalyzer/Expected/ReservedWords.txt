Lexeme                                 Token                          Attribute
-------------------------------------------------------------------------------
if									   IfToken				
else								   ElseToken
while								   WhileToken
float								   FloatToken
int									   IntToken
char								   CharToken
break								   BreakToken
continue							   ContinueToken
void								   VoidToken
if0else                                IdentifierToken                
0                                      NumberToken                    0
else                                   ElseToken  
ifelse                                 IdentifierToken     
if2                                    IdentifierToken
const                                    ConstToken     
x										IdentifierToken     